README.md

```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
 docker-compose -f "docker-compose.yml" up -d --build
 ```
 
-ps: If you have this kind of error “no matching manifest for windows/amd64 in the manifest list entries”, switch to Linux container in Docker Desktop.
+P.S. If you have the kind of error "no matching manifest for windows/amd64 in the manifest list entries", switch to **Linux container** in Docker Desktop.
```

Comment:
P.S. If the kind of error "no matching manifest for windows/amd64 in the manifest list entries" occurs, switch to **Linux container** in Docker Desktop.